frontend/src/features/settings/components/creditAccount/StatusChip.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  CreditAccountStatusType,
+  CreditAccountStatusDisplayValues,
+} from "../../types/creditAccount";
+import "./StatusChip.scss";
+
+interface StatusChipProps {
+  status: CreditAccountStatusType | "SUSPENDED";
```

Comment:
Why not just add "SUSPENDED" as part of the status types enum?

Reply:
@zgong-gov This is because "SUSPENDED" is not a valid Credit Account status type returned from the API, however the styling for the chip that shows the user if a company is suspended shares the same styling as the StatusChip component that displays the credit account status to the user. Rather than create two separate components based on this logic (e.g. CreditAccountStatusChip and SuspendedStatusChip), I prefer to have one shared component like this.

Reply:
ok